src/vendor/command/l2cap.rs

```diff
@@ -436,20 +429,22 @@ impl L2CapCocFlowControl {
 #[derive(Debug, Clone, Copy)]
 #[cfg_attr(feature = "defmt", derive(defmt::Format))]
 /// Parameter for the [coc_tx_data](L2capCommands::coc_tx_data) command
-pub struct L2CapCocTxData {
+pub struct L2CapCocTxData<'a> {
     pub channel_index: u8,
     pub length: u16,
-    pub data: [u8; 252],
+    /// Value to be written. The maximum length is 248 bytes.
+    pub data: &'a [u8],
 }
 
-impl L2CapCocTxData {
+impl<'a> L2CapCocTxData<'a> {
+    const MIN_LENGTH: usize = 4;
     const MAX_LENGTH: usize = 256;
 
     fn copy_into_slice(&self, bytes: &mut [u8]) {
         assert!(bytes.len() >= Self::MAX_LENGTH);
```

Comment:
It looks like we should be asserting for `MIN_LENGHT`?

Reply:
Since this is a static assertion, the byte array we are copying into must be large enough to support the maximum payload size.

Reply:
Never mind... It is not a static assertion and we use min/max elsewhere...